src/app/page.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import CampaignHero from "./components/hero/campaignHero";
+import HomePageMainContent from "./components/homePageMainContent/homePageMainContent";
+import ExploreFurther from "./components/exploreFurther/exploreFurther";
+
+import { TemplateAppContainer } from "@nypl/design-system-react-components";
+import React from "react";
+
+export default function Page(props: any) {
```

Comment:
Same idea as the About page route. Could this be a server component if it renders another new component such as `<HomePage />` that contains all the DS-related component code.